src/t8_schemes/t8_default/t8_default_pyramid/t8_default_pyramid_cxx.hxx

```diff
@@ -143,10 +143,7 @@ public:
 /** Construct the nearest common ancestor of two elements in the same tree. */
```

Comment:
Add doxygen documentation like you did in [t8_dpyramid_bits.h](https://github.com/holke/t8code/pull/284/files#diff-a9bfcd11510af8bd04f6451e034b1a177237f9bdc41eca8b3f86eb4b3fbd53bf)